### PR TITLE
added logger

### PR DESCRIPTION
**Pull request by Martynice**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/impl/UserDaoImpl.java [hunk 1]
```diff
@@ -6,20 +6,25 @@
...
     public User add(User user) {
+        logger.info("Trying to add user " + user);
         Transaction transaction = null;
         Session session = null;
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
             session.save(user);
             transaction.commit();
+            logger.info("User hall was successfully added");
```
Comment (tarkaiev) [inline src/main/java/com/dev/cinema/dao/impl/UserDaoImpl.java]: user hall?

File: src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 2]
```diff
@@ -6,20 +6,25 @@
...
 import org.hibernate.Session;
 import org.hibernate.Transaction;
 
 @Dao
 public class ShoppingCartDaoImpl implements ShoppingCartDao {
+    private static final Logger logger = Logger.getLogger(ShoppingCartDaoImpl.class);
+
     @Override
     public ShoppingCart add(ShoppingCart shoppingCart) {
+        logger.info("Trying to add shopping cart" + shoppingCart);
```
Comment (tarkaiev) [inline src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java]: space here

File: src/main/java/com/dev/cinema/dao/impl/OrderDaoImpl.java [hunk 3]
```diff
@@ -36,6 +41,7 @@ public Order add(Order order) {
 
     @Override
     public List<Order> getOrderHistory(User user) {
+        logger.info("Trying to get order history");
```
Comment (tarkaiev) [inline src/main/java/com/dev/cinema/dao/impl/OrderDaoImpl.java]: add some info about user

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 4]
```diff
@@ -6,21 +6,26 @@
...
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger logger = Logger.getLogger(ShoppingCartDaoImpl.class);
+
     @Override
     public CinemaHall add(CinemaHall cinemaHall) {
+        logger.info("Trying to add cinema hall" + cinemaHall);
```
Comment (tarkaiev) [inline src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java]: space

**Review by tarkaiev — COMMENTED** [review]
well done!

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 5]
```diff
@@ -6,21 +6,26 @@
...
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 
 @Dao
 public class CinemaHallDaoImpl implements CinemaHallDao {
+    private static final Logger logger = Logger.getLogger(ShoppingCartDaoImpl.class);
+
     @Override
     public CinemaHall add(CinemaHall cinemaHall) {
+        logger.info("Trying to add cinema hall " + cinemaHall);
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java]: let's remove this, i think that success  message would be enough

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 6]
```diff
@@ -36,6 +41,7 @@ public CinemaHall add(CinemaHall cinemaHall) {
 
     @Override
     public List<CinemaHall> getAll() {
+        logger.info("Trying to get add cinema halls");
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java]: same

File: src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java [hunk 7]
```diff
@@ -6,20 +6,25 @@
...
     public ShoppingCart add(ShoppingCart shoppingCart) {
+        logger.info("Trying to add shopping cart " + shoppingCart);
         Transaction transaction = null;
         Session session = null;
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
             session.save(shoppingCart);
             transaction.commit();
+            logger.info("Shopping cart was successfully added");
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java]: let's add shoppingCart id to logger message

**Review by dmytro-hryhoruk — COMMENTED** [review]
Apply suggestions to all dao layers

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java (hunk 5 above)
Comment (Martynice) [reply src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java]: done

File: src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java (hunk 7 above)
Comment (Martynice) [reply src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**